plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostingFun/Core.hs

```diff
@@ -437,6 +483,21 @@ runThreeArgumentModel
     (ModelThreeArgumentsLinearInZ (OneVariableLinearFunction intercept slope)) =
         lazy $ \_ _ costs3 ->
             scaleLinearly intercept slope costs3
+runThreeArgumentModel
+    (ModelThreeArgumentsQuadraticInZ f) =
+        lazy $ \_ _ costs3 -> CostLast $ evaluateOneVariableQuadraticFunction f $ sumCostStream costs3
```

Comment:
Same. I guess we need a Note. And we should reference it in old costing functions too. I should've done that myself when I introduced lazy costing. Feel free to do it here or create a ticket for me or yourself.

Reply:
Since we're talking about laziness, I made a ticket for you to do it at PLT-9312.

Reply:
Thanks!